### PR TITLE
Misc doc fixes

### PR DESCRIPTION
**Pull request by sjspielman**
This PR fixes some miscellaneous typos and such, _including one 404'd link_!, in the docs. These are just what I noticed when reviewing docs in prep for presenting the ScPCA poster.

### CODE REVIEW

**Comment by sjspielman** [comment]
Also I want to note that there's a lot of back and forth between uppercase vs lowercase `alevin-fry` and "Alevin fry." We might want to clean this up here, too.

File: docs/faq.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ We also compared the mean gene expression reported for each gene by both methods
 ![](https://github.com/AlexsLemonade/alsf-scpca/blob/c0c2442d7242f6e06a5ac6d1e45bd1951780da14/analysis/docs-figures/plots/gene_exp_correlation.png?raw=true)
 
 Recent reports from others support our findings.
-[He _et al._ (2021)](https://doi.org/10.1101/2021.06.29.450377)) demonstrated that Alevin-fry can process single-cell and single-nuclei data more quickly and efficiently then other available methods, while also decreasing the false positive rate of gene detection that is commonly seen in methods that utilize transcriptome alignment.
+[He _et al._ (2021)](https://doi.org/10.1101/2021.06.29.450377) demonstrated that Alevin-fry can process single-cell and single-nuclei data more quickly and efficiently then other available methods, while also decreasing the false positive rate of gene detection that is commonly seen in methods that utilize transcriptome alignment.
```
Comment (allyhawkins) [inline docs/faq.md:20]:
```suggestion
[He _et al._ (2021)](https://doi.org/10.1101/2021.06.29.450377) demonstrated that `alevin-fry` can process single-cell and single-nuclei data more quickly and efficiently then other available methods, while also decreasing the false positive rate of gene detection that is commonly seen in methods that utilize transcriptome alignment.
```
Looking at the publication of alevin-fry they use lower case, so let's go with that. Also while we are doing this we might want to have all of them consistent in that we put `alevin-fry` in code formatted text.

**Review by allyhawkins — APPROVED** [review]
Just one comment but other than that looks good. Thanks for doing this!

File: docs/faq.md (hunk 1 above)
Comment (sjspielman) [reply docs/faq.md:20]: Ok, I can do a pass throughout the docs for this!

**Pull request merged**